tests/runtime/json-output

```diff
@@ -178,3 +178,10 @@ EXPECT {"type": "value", "data": 333}
 NAME enum_symbolized_printf
 RUN {{BPFTRACE}} -q -f json -e 'enum Foo { ONE = 1, TWO = 2, OTHER = 99999 }; BEGIN { printf("%d %s %d %s %d %s\n", ONE, ONE, TWO, TWO, OTHER, OTHER); exit() }'
 EXPECT {"type": "printf", "data": "1 ONE 2 TWO 99999 OTHER\n"}
+
+# Just check that the json is well formatted
+# Note: if we fix the underlying issue we might need to change/delete this test
+NAME none type
+RUN {{BPFTRACE}} -q -f json -e 'kprobe:vfs_write {print((*(struct file *)arg0).f_owner); exit()}' | python3 -c 'import sys,json; print(json.load(sys.stdin)["type"])'
```

Comment:
kprobe:vfs_write is not guaranteed to fire, especially on a quiet host. Might need a call to `testprog/syscall`. Or a more reliable attachpoint.

Reply:
Good point. Let me add a syscall in a test prog.